garglk/cheapglk/cgfref.cpp

```diff
@@ -38,8 +38,17 @@
 #include <sys/stat.h> /* for stat() */
 
 #ifdef _WIN32
+#define _CRT_INTERNAL_NONSTDC_NAMES 1
+#define R_OK 02
```

Comment:
This is a redefinition on MinGW, as it already provides `R_OK`, so this ought to at least check whether it's defined first.

Reply:
Also, not sure of the effect it'd have, but on MinGW (at least the version I have installed), `R_OK` has the value 4, not 2, so there's a mismatch.

Reply:
Yeah, this one is plain wrong. `02` would check for writing rather than reading.

It's also redundant with the definition I later added to my minimal `unistd.h` so it ought to go anyways.